setup.py

```diff
@@ -97,6 +97,7 @@
     'requests~=2.0,>=2.15.1',
     'timeout-decorator~=0.0,>=0.3.3',
     'Babel~=2.0,>=2.4.0',
+    'setproctitle~=1.1,>=1.1.10',
```

Comment:
`~=1.0`, otherwise it will pin `1.1`.

Reply:
I will never understand/remember this bit :+1: